### PR TITLE
[TRANSLATIONS] Lithuanian

### PR DESCRIPTION
**Pull request by Dalgelis**
**Describe Pull request**
This is Lithuanian translation of this script.

This PR is not to fix an issue

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [no] (Be honest)
- Does your code fit the style guidelines? [yes]
- Does your PR fit the contribution guidelines? [yes]

### CODE REVIEW

**Comment by Z3rio** [comment]
You have to follow the Template or include all the information from the template in some way, which this PR saves to do. The default questions are shown upon trying to merge your branch.

**Comment by Dalgelis** [comment]
Done

**Review by Z3rio — APPROVED** [review]
Looks good to me, not tested ingame though. And I dont know lithuanian so if someone knows the language then please give your feedback on if its good or not :)

File: locale/lt.lua [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    error = {
+        dealer_decline = "Pardavėjas atsisako rodyti jums ginklus",
+        talk_cop = "Kalbėkis su teisėsauga, dėl ginklų licensijos"
+    },
+    success = {
+        dealer_verify = "Pardavėjas patvirtino jūsų licensija"
+    },
+}
+
+Lang = Locale:new({phrases = Translations})
```
Comment (tom-osborne) [inline locale/lt.lua:15]: This needs to handle the `qb_locale` convar for auto-switching locales. Please use the following code:
```lua
if GetConvar('qb_locale', 'en') == 'lt' then
    Lang = Locale:new({
        phrases = Translations,
        warnOnMissing = true,
        fallbackLang = Lang,
    })
end
```

**Review by tom-osborne — CHANGES_REQUESTED** [review]

**Pull request merged**